doc/source/_res/rn/useful-links.txt

```diff
@@ -0,0 +1,10 @@
+Useful links
+========================
+
+- The installation instructions for `Percona Server for MySQL <https://www.percona.com/doc/percona-server/LATEST/installation.html>`__ are available.
```

Comment:
I would change it to smth like:
This is the instruction on how to install `Percona Server for MySQL <https://www.percona.com/doc/percona-server/LATEST/installation.html>`__ .

Reply:
Updated.